x-pack/plugins/features/server/oss_features.test.ts

```diff
@@ -86,7 +86,8 @@ Array [
             new KibanaFeature(featureConfig),
             {
               augmentWithSubFeaturePrivileges: true,
-              licenseType,
+              licenseHasAtLeast: (licenseTypeToCheck: LicenseType) =>
+                LICENSE_TYPE[licenseTypeToCheck] <= LICENSE_TYPE[licenseType],
```

Comment:
I don't love importing `LICENSE_TYPE` directly from the licensing server for this unit test, but it seemed to be the least-bad option here.